jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -205,10 +205,16 @@ void jbmc_parse_optionst::get_command_line_options(optionst &options)
     }
   }
 
+  // generate unwinding assertions
+  options.set_option(
+    "unwinding-assertions",
+    cmdline.isset("unwinding-assertions") ||
```

Comment:
Since this is a breaking change, wouldn't it be better to remove --unwinding-assertions entirely?

Reply:
:question: